os_stub/mbedtlslib/include/mbedtls/config.h

```diff
@@ -3749,6 +3779,12 @@ extern int my_snprintf(char *str, size_t size, const char *format, ...);
  */
 /*#define MBEDTLS_PSA_KEY_SLOT_COUNT 32*/
 
+/* RSA OPTIONS */
+/** Minimum RSA key size that can be generated in bits (Minimum possible value
+ * is 128 bits)
+ */
+#define MBEDTLS_RSA_GEN_KEY_MIN_BITS        512
```

Comment:
For SPDM, the RSA MIN_BIT is 2048.

Reply:
I added this config option as the default value for this in mbedtls is 1024 and in rsa_verify.c
#define LIBSPDM_RSA_MODULUS_LENGTH 512
Not adding this config options fails test_crypt unit test.